### PR TITLE
MX-261: an endpoint that can create access tokens for a member

### PR DESCRIPTION
**Pull request by bystam**
# Jira Issue: [MX-261] 

## What?
- A new endpoint that can create access tokens for an input member

## Why?
- We will want this for our initiative to push member creation to the signing point
- In the onboarding flow, we will want to give an access token to the client upon signing. Afterwards, this token can be used to perform further action (such as connecting payments)

## Optional checklist
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

[MX-261]: https://hedvig.atlassian.net/browse/MX-261

### CODE REVIEW

File: src/main/java/com/hedvig/gateway/AuthController.kt [hunk 1]
```diff
@@ -18,8 +22,26 @@ import org.springframework.web.bind.annotation.RestController
 class AuthController(
     private val repo: AuthorizationRowRepository,
     private val exchangeService: ExchangeService,
+    private val tokenService: TokenService,
     private val config: ServiceAccountTokenConfig
 ) {
+    @PostMapping("/auth")
+    fun createToken(
+        @RequestHeader("token") underwriterToken: String,
```
Comment (rasmusguterstam) [inline src/main/java/com/hedvig/gateway/AuthController.kt:30]: Just curious, why not `Authorization`?

Comment (rasmusguterstam) [reply src/main/java/com/hedvig/gateway/AuthController.kt:30]: As header that is

Comment (bystam) [reply src/main/java/com/hedvig/gateway/AuthController.kt:30]: I just mimicked exactly what the other endpoints did 🤷

Comment (rasmusguterstam) [reply src/main/java/com/hedvig/gateway/AuthController.kt:30]: Fair enough! 😄

**Pull request merged**